src/server/pps/cmds/cmds.go

```diff
@@ -921,6 +921,14 @@ All jobs created by a pipeline will create commits in the pipeline's output repo
 			if len(args) == 0 && !all {
 				return errors.Errorf("either a pipeline name or the --all flag needs to be provided")
 			}
+			if splitTransaction {
+				fmt.Println("Warning: This command with the --split-txn flag must be run to completion / rerun upon failure or Pachyderm will be left in an inconsistent state.")
```

Comment:
We are merging directives in these instructions. Let's split these directives out into more complete ideas to be more clear. Also warning should be in all caps (consistency with other notices like this).

This message should read as follows:

WARNING: If using the --split-txn flag, this command must run until complete. If a failure or incomplete run occurs, then Pachyderm will be left in an inconsistent state. To resolve an inconsistent state, rerun this command.